lib/streamex/config.ex

```diff
@@ -4,18 +4,18 @@ defmodule Streamex.Config do
   for configuring the connection client.
   """
 
-  @api_url "api.getstream.io/api"
-  @default_timeout 3000
+  @api_url "api.stream-io-api.com/api"
```

Comment:
I'm seeing https://chat.stream-io-api.com in the official Go client, so we probably need to update this and regenerate all the VCRs.

https://github.com/GetStream/stream-chat-go/blob/master/client.go#L29